.github/workflows/comment-build-test.yml

```
@@ -0,0 +1,64 @@
+    uses: ./.github/workflows/core-build-test.yml
+    with:
+      runner: ${{ needs.parse-and-trigger.outputs.runner }}
+    secrets: inherit
+
+  comment-result:
+    needs: [parse-and-trigger, run-ci]
+    if: always() && needs.parse-and-trigger.outputs.should_run == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment result
+        uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)